meta/documents/changelog_en.md

```diff
@@ -1,5 +1,13 @@
 # Release Notes for Elastic Export Billiger.de
 
+## v1.0.1 (2017-07-12)
+
+### Changed
+- The plugin Elastic Export is now required to use the plugin format BilligerDE.
+
+### Added
+- Added optional fields in the export csv file to provide the extra properties needed for billiger.de.
```

Comment:
The fields of the CSV file were extended to support new properties for billiger.de.